config/locales/views/admin/en.log_events.yml

```diff
@@ -0,0 +1,18 @@
+en:
+  views:
+    log_events:
+      index:
+        account__create: Payment Source created
```

Comment:
I wonder if it might be easier to think about if we structured this as
```
log_events:
  index: 
    account:
      create: Payment Source created
      update: Payment Source updated
```

And I could even see this moving into the `LogEvent#to_s` or similar. It doesn't quite feel like it's necessarily `index`-specific. But I could see punting on that.